src/lib/holocene/copyable.svelte

```diff
@@ -13,34 +13,51 @@
 
 {#if clickAllToCopy}
   <button
-    class="group flex items-center gap-2 {$$props['container-class']}"
+    class="group flex items-center gap-1 {$$props['container-class']}"
     on:click={(e) => copy(e, content)}
   >
     <slot>
       <span class={$$props.class} class:select-all={!$$slots.default}
         >{content}</span
       >
     </slot>
-    <IconButton
+    <button
       on:click={(e) => copy(e, content)}
```

Comment:
Should we just have a CopyButton component?

Reply:
Yeah, was just talking with @tigernaut about this. I think probably? But I think we likely need to have a bigger discussion of if we want this and `IconButton` to be variants of the `Button` component.